rasa/shared/core/slots.py

```diff
@@ -1,10 +1,10 @@
 import logging
-from typing import Any, Dict, List, NoReturn, Optional, Text, Type
 
-from rasa.core import utils
 from typing import Any, Dict, List, NoReturn, Optional, Text, Type
-from rasa.core.constants import DEFAULT_CATEGORICAL_SLOT_VALUE
-from rasa.utils.common import class_from_module_path, raise_warning
+
+import rasa.shared.utils.common
+from rasa.shared.core.constants import DEFAULT_CATEGORICAL_SLOT_VALUE
+from rasa.shared.utils import io as io_utils, common as common_utils
```

Comment:
well, I think we should choose one name for imported stuff, and use either `import rasa.shared.utils.io as io_utils` or just `import rasa.shared.utils.io`, but not both ways mixed together. 
the reason is that different synonyms for imported modules make it more difficult to search for specific functions and their usage using utils like `grep`

Reply:
in addition, you import `rasa.shared.utils.common` two times here

line 5:
`import rasa.shared.utils.common`

line 7:
`from rasa.shared.utils import io as io_utils, common as common_utils`